linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/tests/correct_stig.pass.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# make existing entries pass
+for acct in $(grep -ve ':\(!!\|\*\):' /etc/shadow | awk -F: '{print $1}'); do
+    chage -M 30 $acct
```

Comment:
In all test scenarios, add the following, `-d $(date +%F)`, so that the test suite can connect without need to reset password.
Full line: `chage -d $(date +%F) -M 30 $acct`.